### DISCUSSION TITLE
Escape sequences

### DISCUSSION THREAD

**Discussion by CCurl** [microsoft/terminal#17471] [Q&A] [answered]
I have so far been unable to find a reference for which set(s) of escape sequences are supported. Is there a page somewhere that has that information?

Thanks

**Comment by Yewzir** [comment, marked as answer]
@CCurl 

Perhaps [this](https://learn.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences) will help.

Kind regards
Yewzir

**Comment by zadjii-msft** [comment]
The doc linked above is probably the most right. It's also probably greatly out of date.

We also were doing something more automatic in https://github.com/microsoft/terminal/pull/1884#issuecomment-518917832, but that also hasn't been updated in a long while. I don't think https://github.com/microsoft/terminal/blob/main/doc/reference/Build-SupportedSequenceIndex.ps1 still works
